config/routes.rb

```diff
@@ -15,7 +15,12 @@
     get 'user', to: 'users#show'
     get 'profile', to: 'users#show'
 
-    resource :education_benefits_claims, only: :create
+    resource :education_benefits_claims, only: [:create] do
+      unless Rails.env.production? && ENV['EDU_FORM_SHOW'] != 'true'
```

Comment:
could you use a constraint here so that the same logic could be reused in the controller? ithttp://guides.rubyonrails.org/routing.html#advanced-constraints